feature-libs/organization/administration/components/unit/list/unit-list.component.ts

```diff
@@ -13,7 +14,14 @@ import { UnitTreeService } from '../services/unit-tree.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class UnitListComponent {
-  constructor(protected unitTreeService: UnitTreeService) {}
+  constructor(
+    protected unitTreeService: UnitTreeService,
+    protected orgUnitService?: OrgUnitService
+  ) {}
+
+  isUpdatingUnitAllowed = this.orgUnitService
```

Comment:
we can use `readonly` to prevent modification after assignment

Reply:
done